### ISSUE TITLE
Can we support Custom CSS Variables?

### ISSUE THREAD

**Issue by Suges** [laosb/SvelteNova#6]
As shown here: https://svelte.dev/repl/bb42c17d585b4236824e1d5e48fe92b5?version=3.38.0

Code like this:
```svelte
  <Checkbox --box-size=12px --top-align=-1px
    bind:checked={user.isAdmin}
    on:click={toggleAdmin}
    label=Admin
    topalign=-1
    tooltip={`Click to give administrator privileges to or remove them from ${user.username}`}
  />
```
...doesn't throw any errors in VS Code, but it throws lint errors in Nova:

<img width="341" alt="Screen Shot 2021-05-02 at 1 20 56 PM" src="https://user-images.githubusercontent.com/7140406/116821796-4dbb1280-ab49-11eb-81ca-10ccf127a002.png">

**Comment by Suges** [comment]
As of Version 7 this now magically works!

**Comment by laosb** [comment]
This is implemented in 3.38 so I assume upgrading language server to latest will fix this. Will do.

**Comment by laosb** [comment]
Just released 0.1.7 bumping to lang server 0.14.6. I tested in my project and it seems to work fine now.

Please try it out and reopen if you are still hitting this bug.

**Comment by Suges** [comment]
Yup working great now! Thanks very much for keeping this plugin updated; love Svelte, love Nova.
